packages/db/src/schema/notification.ts

```diff
@@ -0,0 +1,53 @@
+import { relations, sql } from "drizzle-orm";
+import {
+  integer,
+  primaryKey,
+  sqliteTable,
+  text,
+} from "drizzle-orm/sqlite-core";
+import { createSelectSchema } from "drizzle-zod";
+import { z } from "zod";
+
+import { monitor } from "./monitor";
+import { workspace } from "./workspace";
+
+const name = ["email", "discord", "slack"] as const;
+
+export const notification = sqliteTable("notification", {
+  id: integer("id").primaryKey(),
+  name: text("name", { enum: name }).notNull(),
+  data: text("data", { mode: "json" }).notNull().default("{}"),
```

Comment:
Is `data` where we will store information for the different notifications? Like _email address_ for "email", _webhook url_ for "slack",...